src/azure-cli-core/setup.py

```diff
@@ -67,7 +67,7 @@
 
 # dependencies for specific OSes
 if not sys.platform.startswith('cygwin'):
-    DEPENDENCIES.append('psutil~=5.7')
+    DEPENDENCIES.append('psutil~=5.8')
```

Comment:
Use [psutil 5.8](https://pypi.org/project/psutil/5.8.0/) in order to get Python 3.9 support.